app/model/till.rb

```diff
@@ -0,0 +1,74 @@
+class Till
+  attr_reader :data, :tax, :order_total
+  DEFAULT_TAX = 8.64
+
+  def initialize(data, tax = DEFAULT_TAX)
+    @data = data[0]
+    @tax = tax
+    @order_total = 0
+  end
+
+  def calculate_line_total(line)
+    if line.keys[0].to_s.include?("Muffin")
+      price = (price(line) * 0.9).round(2)
+    else
+      price = price(line)
+    end
+    quantity(line) * price
+  end
+
+  def calculate_base_total(order)
+    order[:items].inject(0) do |total, order_item|
+      total + (calculate_line_total(order_item))
+    end
+      .round(2)
```

Comment:
Style/DotPosition: Place the . on the previous line, together with the method call receiver. (https://github.com/bbatsov/ruby-style-guide#consistent-multi-line-chains)